device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/port_config.ini

```diff
@@ -0,0 +1,21 @@
+# name         lanes             alias        index  role       speed       asic_port_name     coreId      corePortId      numVoq
+Ethernet0      72,73,74,75       Ethernet1/1  1      Ext        100000      Eth0-ASIC0         1           1               8
+Ethernet8      80,81,82,83       Ethernet2/1  2      Ext        100000      Eth8-ASIC0         1           3               8
+Ethernet16     88,89,90,91       Ethernet3/1  3      Ext        100000      Eth16-ASIC0        1           5               8
+Ethernet24     96,97,98,99       Ethernet4/1  4      Ext        100000      Eth24-ASIC0        1           7               8
+Ethernet32     104,105,106,107   Ethernet5/1  5      Ext        100000      Eth32-ASIC0        1           9               8
+Ethernet40     112,113,114,115   Ethernet6/1  6      Ext        100000      Eth40-ASIC0        1           11              8
+Ethernet48     120,121,122,123   Ethernet7/1  7      Ext        100000      Eth48-ASIC0        1           13              8
+Ethernet56     128,129,130,131   Ethernet8/1  8      Ext        100000      Eth56-ASIC0        1           15              8
+Ethernet64     136,137,138,139   Ethernet9/1  9      Ext        100000      Eth64-ASIC0        1           17              8
+Ethernet72     64,65,66,67       Ethernet10/1 10     Ext        100000      Eth72-ASIC0        0           19              8
+Ethernet80     56,57,58,59       Ethernet11/1 11     Ext        100000      Eth80-ASIC0        0           21              8
+Ethernet88     48,49,50,51       Ethernet12/1 12     Ext        100000      Eth88-ASIC0        0           23              8
+Ethernet96     40,41,42,43       Ethernet13/1 13     Ext        100000      Eth96-ASIC0        0           25              8
+Ethernet104    32,33,34,35       Ethernet14/1 14     Ext        100000      Eth104-ASIC0       0           27              8
+Ethernet112    24,25,26,27       Ethernet15/1 15     Ext        100000      Eth112-ASIC0       0           29              8
+Ethernet120    16,17,18,19       Ethernet16/1 16     Ext        100000      Eth120-ASIC0       0           31              8
+Ethernet128    8,9,10,11         Ethernet17/1 17     Ext        100000      Eth128-ASIC0       0           33              8
+Ethernet136    0,1,2,3           Ethernet18/1 18     Ext        100000      Eth136-ASIC0       0           35              8
+Ethernet-Rec0  221               Recirc0/0    19     Rec        400000      Rcy0-ASIC0         0           221             8
```

Comment:
The index for Ethernet-Rec0 and Ethernet-IB0 is overlapping with the index for Ethernet144 and Ethernet-152

Reply:
Fixed